js/example.js

```diff
@@ -96,3 +124,12 @@ function stripScripts(source) {
   }
   return div.innerHTML;
 }
+
+function setClipboard(value) {
+  const tempTextarea = document.createElement("textarea");
+  tempTextarea.value = value;
+  document.body.appendChild(tempTextarea);
+  tempTextarea.select();
+  document.execCommand("copy");
```

Comment:
I think you need a try catch here to stop a console error and allow you to provide a useful console.warning.